src/swagger-inline.js

```diff
@@ -94,18 +94,22 @@ function swaggerInline(globPatterns, providedOptions) {
               return _.isEmpty(s);
             });
             schemas = _.concat(schemas, scheme);
+            return Promise.resolve()
           } catch (e) {
-            throw new Error(e.toString(), fileInfo.fileName);
+            return Promise.reject(new Error(e.toString(), fileInfo.fileName))
           }
-        });
-
-        log(`${endpoints.length} definitions found...`);
-        log(`${schemas.length} schemas found...`);
-
-        const baseObjWithEndpoints = mergeEndpointsWithBase(baseObj, endpoints);
-        const swagger = mergeSchemasWithBase(baseObjWithEndpoints, schemas);
-
-        return outputResult(swagger, options);
+        })).then(() => {
+          log(`${endpoints.length} definitions found...`);
+          log(`${schemas.length} schemas found...`);
+
+          const baseObjWithEndpoints = mergeEndpointsWithBase(baseObj, endpoints);
+          const swagger = mergeSchemasWithBase(baseObjWithEndpoints, schemas);
+
+          return outputResult(swagger, options);
+        }).catch(e => {
+          console.error(e);
```

Comment:
Thoughts on throwing the error instead of putting it in the console?

Reply:
yep throw the error will prevent writing error stack into target file and exit process with non-zero exit code
In previous version. u will get code zero but the generated file contains error message

Reply:
yep updated.
throwing it will be better than exit process directly.

Toolchains not triggers because I edited it on github web page lol. Thought it will be a small fix but found it wrong😆.

Btw wrote a test case to cover process exit code.